src/status_im/utils/deprecated_types.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.utils.types
+(ns status-im.utils.deprecated-types
```

Comment:
Is it possible to add metadata `{:deprecated true :superseded-by "utils.transforms"}` to this ns as an additional measure to prevent devs from using this namespace?

Reply:
thanks